lambda/app/src/utils/helpers.ts

```diff
@@ -132,10 +133,22 @@ export const getWhereClauseForAllRequests = (data: {
     where.archived = archiveStatus[0] === 'archived';
   }
 
-  if (realms)
+  // silver and gold IDPs are in different columns requiring an `and or` query
+  if (realms && !devIdps) {
```

Comment:
@thegentlemanphysicist the `devIdps` would always be non-empty as selecting an idp is mandatory to submit integration request